### ISSUE TITLE
Daily builds of the JS repo don't seem to be publishing

### ISSUE THREAD

**Issue by cleemullins** [Microsoft/botbuilder-js#525]
The [daily build feed](https://botbuilder.myget.org/feed/Packages/botbuilder-v4-js-daily) doesn't appear to have been updated recently. 

The daily builds that push code to the MyGet feed may have something wrong with them.

**Comment by stevengum** [comment]
Per the latest CRON job:

https://travis-ci.org/Microsoft/botbuilder-js/builds/439369250#L1497-L1505:

```bash
$ test $TRAVIS_EVENT_TYPE = "cron" && echo Begin deploy
  && npm publish libraries/botbuilder --registry=https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
  && npm publish libraries/botbuilder-ai --registry=https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
  && npm publish libraries/botbuilder-azure --registry=https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
  && npm publish libraries/botbuilder-core --registry=https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
  && npm publish libraries/botbuilder-dialogs --registry=https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
  && npm publish libraries/botframework-config --registry=https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
  && npm publish libraries/botframework-connector --registry=https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
  && npm publish libraries/botframework-schema --registry=https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
  && echo End deploy
Begin deploy
npm ERR! registry error parsing json
npm ERR! publish Failed PUT 403
npm ERR! Unexpected token Y in JSON at position 0
npm ERR! You are not allowed to access this feed.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-10-09T21_51_34_253Z-debug.log
```

**Comment by stevengum** [comment]
The last cron build that finished and published is #1584: https://travis-ci.org/Microsoft/botbuilder-js/builds/431729689

The first cron build after that failed is #1587: https://travis-ci.org/Microsoft/botbuilder-js/builds/431980832

**Comment by cleemullins** [comment]
Working on this now. Possibly fixed in PR #542

**Comment by cleemullins** [comment]
Fix pending against master in #543

**Comment by cleemullins** [comment]
Fixed.
